src/components/app.jsx

```diff
@@ -18,6 +18,16 @@ class App extends React.Component {
     this.addCards = this.addCards.bind(this);
   }
 
+  componentDidMount(){
+    let cards = localStorage.getItem('flash-cards');
+    cards = JSON.parse(cards);
+    if(cards !== null){
+        this.setState({
+            cards: [cards]
```

Comment:
This looks like it would wrap your cards in an extra array. Does this work? I'm unsure cause you have something weird down below with `localStorage`. I would recommend removing the extra square brackets around `cards`.